.maestro/shared/pre_onboarding.yaml

```diff
@@ -11,5 +11,8 @@ appId: com.duckduckgo.mobile.android
     direction: DOWN
 - tapOn: "choose your browser"
 - tapOn: "cancel"
+- assertVisible:
```

Comment:
also need to change: `text: ".*Ready for a better, more private internet?.*”` with the new string